db/migrate/20231129042355_create_posts.rb

```diff
@@ -4,8 +4,9 @@ def change
       t.references :author, foreign_key: { to_table: :users }
       t.string :title
       t.string :text
-      t.integer :likes_counter
-      t.integer :comments_counter
+      t.integer :likes_counter, default: 0
+      t.integer :comments_counter, default: 0
+      t.references :author, foreign_key: { to_table: users }
```

Comment:
- [ ] This line is repeated at the top of the file which causes error. Please revert the schema back, remove it, then migrate again. 🙏